tools/devctr/Dockerfile.aarch64

```diff
@@ -7,12 +7,14 @@ FROM ubuntu:18.04
 ARG RUST_TOOLCHAIN="1.52.1"
 ARG TINI_VERSION_TAG="v0.18.0"
 ARG TMP_BUILD_DIR=/tmp/build
-ARG TMP_POERTY_DIR=/tmp/poetry
+ARG TMP_POETRY_DIR=/tmp/poetry
 ARG FIRECRACKER_SRC_DIR="/firecracker"
 ARG FIRECRACKER_BUILD_DIR="$FIRECRACKER_SRC_DIR/build"
 ARG CARGO_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_registry"
 ARG CARGO_GIT_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_git_registry"
 ARG DEBIAN_FRONTEND=noninteractive
+# By default we don't provide a poetry.lock file
+ARG POETRY_LOCK_PATH="/dev/null/*"
```

Comment:
I don't understand this.. there is no `/dev/null/*`, `/dev/null` is not a dir entry.

Reply:
The idea is to provide a path that guarantees that no file will be copied. This is a workaround since there is no good way to execute a conditional `COPY` in the dockerfile. So instead we provide a regex that leads to an empty list of files. I also tried to simply provide an empty string, but that leads to a freeze. Personally I couldn't find a better option. I can add a more detailed comment though.

Reply:
ok, lgtm!